stackBr/StackBr.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include "Stack.h"
+#include "StackBr.h"
+
+void freeStack(struct Stack* stack) 
+{ 
+    Node* current = stack->head;
+    Node* next;
+    while (current != NULL) {
+        next = current->next;
+        free(current);
+        current = next;
+    }
+    stack->head = NULL;
+}
+
+int isMatching(char char1, char char2) 
+{
+    if (char1 == '(' && char2 == ')') return 1;
+    if (char1 == '[' && char2 == ']') return 1;
+    if (char1 == '{' && char2 == '}') return 1;
+    return 0;
+}
+
+int isBalanced(char* str) 
+{
+    struct Stack stack = new(); 
+    int i = 0;
+    for (i = 0; str[i] != '\0'; i++) {
```

Comment:
А это нужно?
Чем плохо такое?
```suggestion
    for (int i = 0; str[i] != '\0'; i++) {
```